source/components/organisms/SummaryList/SummaryListItem.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable no-nested-ternary */
+import React from 'react';
+import { View } from 'react-native';
+import { TouchableHighlight } from 'react-native-gesture-handler';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+import { Input, Text, Icon } from '../../atoms';
+import colors from '../../../styles/colors';
+import { Item } from './SummaryList';
+import DateTimePickerForm from '../../molecules/DateTimePicker/DateTimePickerForm';
+
+const ItemWrapper = styled(View)`
+  flex-direction: row;
+  align-items: flex-end;
+  height: 46px;
+`;
+const InputWrapper = styled.View`
+  align-items: center;
+  justify-content: flex-end;
+  flex: 1;
+  padding-left: 50px;
+`;
+const SmallInput = styled(Input)`
+  height: 40px;
+  padding-top: 8px;
+  padding-bottom: 8px;
+  background-color: transparent;
+  border: none;
+  border-bottom-width: 1px;
+  border-color: black;
+`;
+const SmallText = styled(Text)`
+  height: 40px;
+  font-size: 14px;
+  padding-top: 11px;
+  padding-bottom: 8px;
+  padding-left: 17px;
+`;
+const DeleteButton = styled(Icon)`
+  padding-top: 5px;
+  padding-left: 0px;
+  padding-right: 0px;
+  margin-left: 0px;
+  margin-right: 0px;
+  margin-bottom: 15px;
+  color: black;
+`;
```

Comment:
Would prefix all of these with SummaryItem so that they do not get confused with other components in the code base

Reply:
Since they are not exported, I would say that this makes it way less readable. There is such a thing as too long names.

Reply:
Renamed to add prefixes.